core/rollup_fee.go

```diff
@@ -0,0 +1,22 @@
+package core
+
+import (
+	"math/big"
+)
+
+/// ROLLUP_BASE_TX_SIZE is the encoded rollup transaction's compressed size excluding
+/// the variable length data.
+/// Ref: https://github.com/ethereum-optimism/contracts/blob/409f190518b90301db20d0d4f53760021bc203a8/contracts/optimistic-ethereum/OVM/precompiles/OVM_SequencerEntrypoint.sol#L47
+var ROLLUP_BASE_TX_SIZE int = 96
```

Comment:
This will likely need to change due to https://github.com/ethereum-optimism/contracts/pull/300
Also what do you think about making this a `const`?

Reply:
:+1: if that PR gets merged first, I'll adjust it. This means I'll have to serialize the transaction each time, instead of assuming that the data is constant size right?

Reply:
Yup, will need to `rlp.EncodeToBytes` and use the length of that - https://github.com/ethereum-optimism/go-ethereum/blob/2e468279837326ed882c9a2ea11ccebf1a17b60d/rlp/encode.go#L70